pkgs/top-level/all-packages.nix

```diff
@@ -3060,6 +3060,9 @@ in
 
   doom-bcc = callPackage ../games/zdoom/bcc-git.nix { };
 
+  # syntax warnings on 3.8
```

Comment:
I think warnings are fine?

Reply:
Not really — they produce SyntaxWarnings in log, and that's plain bad ux.
Also, several other derivations are downgrading python versions due to that. 

I will PR them a fix later

Reply:
Most users will never see those warnings. If it works/pass tests with Python 3.8 it's better to have it support more rather than less Python versions, no?

To be clear, what I mean is that the vast majority of users will get this from cache, and not build it themselves, and so issuing warnings during build doesn't really do anything to their UX.

Reply:
But that's Python — they will get warnings every run.

Reply:
I am for using the default `python3Packages` here.